server/channels/api4/user_test.go

```diff
@@ -3831,11 +3831,11 @@ func TestResetPassword(t *testing.T) {
 	resp, err = th.Client.ResetPassword(context.Background(), "junk", "newpwd")
 	require.Error(t, err)
 	CheckBadRequestStatus(t, resp)
-	code := ""
+	var code strings.Builder
 	for range model.TokenSize {
-		code += "a"
+		code.WriteString("a")
 	}
-	resp, err = th.Client.ResetPassword(context.Background(), code, "newpwd")
+	resp, err = th.Client.ResetPassword(context.Background(), code.String(), "newpwd")
```

Comment:
Part of the new checks is a string builder one. (See https://pkg.go.dev/golang.org/x/tools@v0.38.0/go/analysis/passes/modernize#hdr-Analyzer_stringsbuilder)

While the performance improvements are not needed for the test, I'm leaning toward not adding a linter exception as that adds complexity to the config file.